packages/api-openfin/screen-snippet.js

```diff
@@ -0,0 +1,14 @@
+/* globals createImageBitmap, html2canvas */
```

Comment:
Possibly worth adding `createImageBitmap` into `eslintrc.json` as its declared as global in more than 1 file?